docs/source/guides/libraries.rst

```diff
@@ -67,12 +67,256 @@ Python library
 Where to get
 ^^^^^^^^^^^^
 
+Prerequirements:
+
+- cmake(3.11 or higher)
+- git
+- g++
+- boost(1.65 or higher, only system and filesystem)
+- swig(3.0.12 can be built --without-pcre)
+- protobuf
+
+Install iroha python libraries:
```

Comment:
Please use headings
https://sphinx-rtd-theme.readthedocs.io/en/latest/demo/structure.html#structural-elements see example and raw code on GitHub